### ISSUE TITLE
Emerge Medstaffing - IL State Dept received 2021 data

### ISSUE THREAD

**Issue by sliu1000** [CDCgov/prime-reportstream#6363]
## Problem statement

Nicholas McGee from DoIT Illinois recently recieved files from 2021 COVID results.  This is hindering our ability to finish 2021 year end close outs.

## What you need to know

ORC|RE|a480a48c-3ef2-4a95-a87a-d28a220e145a^Emerge Medstaffing L^14D2185111^CLIA|a480a48c-3ef2-4a95-a87a-d28a220e145a^Emerge Medstaffing L^14D2185111^CLIA|||||||||^Watson^Dr. Richard||(312)206-1064^WPN^PH^^1^312^2061064|20211223180000-0000||||||Emerge Medstaffing LLC^L|3330 W 177th St^^Hazel Crest^IL^60429^^^^17031|(888)855-2995^WPN^PH^^1^888^8552995|^^^^60429

OBR|1|a480a48c-3ef2-4a95-a87a-d28a220e145a^Emerge Medstaffing L^14D2185111^CLIA|a480a48c-3ef2-4a95-a87a-d28a220e145a^Emerge Medstaffing L^14D2185111^CLIA|97097-0^SARS-CoV-2 (COVID-19) Ag [Presence] in Upper respiratory specimen by Rapid immunoassay^LN^^^^2.70|||20211223174500-0000|20211223174500-0000||||||||^Watson^Dr. Richard|(312)206-1064^WPN^PH^^1^312^2061064|||||20220808175431-0000|||F

OBX|1|CWE|97097-0^SARS-CoV-2 (COVID-19) Ag [Presence] in Upper respiratory specimen by Rapid immunoassay^LN^^^^2.70||260373001^Detected^SCT|||A^Abnormal (applies to non-numeric results)^HL70078^^^^2.7|||F|||20211223174500-0000|14D2185111^Emerge Medstaffing LLC^CLIA||INDICAID COVID-19 Rapid Antigen Test_PHASE Scientific International, Ltd._EUA^INDICAID COVID-19 Rapid Antigen Test^99ELR^^^^2.70^^INDICAID COVID-19 Rapid Antigen Test_PHASE Scientific International, Ltd._EUA||20211223180000-0000||||Emerge Medstaffing LLC^^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^14D2185111|3330 W 177th St^^Hazel Crest^IL^60429^^^^17031

SPM|1|a480a48c-3ef2-4a95-a87a-d28a220e145a&Emerge Medstaffing L&14D2185111&CLIA^a480a48c-3ef2-4a95-a87a-d28a220e145a&Emerge Medstaffing L&14D2185111&CLIA||697989009^Anterior nares swab^SCT^^^^2.67||||53342003^Internal nose structure (body structure)^SCT^^^^2020-09-01|||||||||20211223174500-0000|20211223174506-0000

## Acceptance criteria

- Please confirm that this data is valid and the reason for the delayed reporting.
- In the future, please provide a notice or send out a mass email to the users submitting data and let them know that they should not be sending COVID data from 2021 as we are working on year end close outs?

## To do

- Assign to Engineer

**Comment by brick-green** [comment]
Reached out to Nathan on this to confirm is it was entered correctly. I mentioned in an email though that I disagree with IL on this. I don't think we should instruct any sender to stop reporting data just because it's old. But I could see us/experience finding value in client research to understand why results are delayed so long in reporting.

cc @sarahnw @BerniXiongA6

**Comment by brick-green** [comment]
Zedd confirmed that this result was backdated. SimpleReport allows tests to be backdated to Jan 2020. I just asked if they can connect with the original submitter to understand why the reporting was delayed

**Comment by brick-green** [comment]
Talked to SimpleReport and they said it's hard to accidentally enter an old test result. There is a warning when anyone tries to submit an old result.  
![image.png](https://images.zenhubusercontent.com/60d23d7dcaf07751be9205b6/4a6fa458-aec9-4fef-9556-5fbad965b5c1)

@sarahnw What do you think about setting our policy that senders should report results as quickly as possible but should not hesitate to report old results?

**Comment by sarahnw** [comment]
I chatted with Megan Light who has recommended that we would not tell a lab to not report/correct it. They'll want to have correct information in their systems which would require sending to the PHD. She also recommends against filtering these old results out - "they are true results and should be passed along to the health department"

Waiting to hear if she's found any documentation that we can point to to back that up. She did confirm that from the CDC's perspective no Covid data has been closed out yet and they wouldn't tell labs not to report or filter out older results. If she passes along some documentation I'll send to IL. If none comes by today though, I'll just follow up with the info.
